package.json

```diff
@@ -127,5 +127,6 @@
     "@chainsafe/blst": {
       "optional": true
     }
-  }
+  },
+  "packageManager": "yarn@1.22.21+sha256.dbed5b7e10c552ba0e1a545c948d5473bc6c5a28ce22a8fd27e493e3e5eb6370"
```

Comment:
Silly question, but why did you add this? Was there a package resolution issue or something?

Reply:
My bad, loca `yarn` keeps adding that